src/test/java/org/gridsuite/study/server/utils/TestUtils.java

```diff
@@ -165,6 +165,8 @@ public static void assertServerRequestsEmptyThenShutdown(MockWebServer server) t
     public static void assertWiremockServerRequestsEmptyThenShutdown(WireMockServer wireMockServer) throws UncheckedInterruptedException, IOException {
         try {
             wireMockServer.checkForUnmatchedRequests(); // requests no matched ? (it returns an exception if a request was not matched by wireMock, but does not complain if it was not verified by 'verify')
+            var req = wireMockServer.findAll(WireMock.anyRequestedFor(WireMock.anyUrl())); // requests no verified ?
```

Comment:
Forgot to clean also ?
Anyway you can remove the duplicated line and reuse `req` below